packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -30,6 +31,9 @@ export function buildPlugin({
 		},
 		resolve(outdir),
 		{
+			bundle: true,
+			additionalModules: [],
+			moduleCollector: noopModuleCollector,
```

Comment:
If we were previously `disableModuleCollection: false` (i.e. we _were_ doing module collection), is the `noopModuleCollector` the correct thing to do here?

Reply:
Good question...
I think that if `rules` is empty and `collectAdditionalModules` is false then the module collector basically does nothing.
But equally we have a moduleCollector in the buildWorker.ts file. So perhaps we should just stick one in here too...?

Reply:
Added in the module collector. Nothing broke. Nothing seems to have changed. Merging.